rmb-sdk-go/peer/peer.go

```diff
@@ -130,38 +140,32 @@ func generateSecureKey(identity substrate.Identity) (*secp256k1.PrivateKey, erro
 }
 
 // getRelayConnections tries to connect to all relays and returns only the successful ones
+// getRelayConnections returns InnerConnections for all valid relay URLs
 func getRelayConnections(relayURLs []string, identity substrate.Identity, session string, twinID uint32) ([]string, []InnerConnection, error) {
-	var successfulRelayURLs []string
-	var successfulConnections []InnerConnection
+	var validRelayURLs []string
+	var connections []InnerConnection
 
 	for _, relayURL := range relayURLs {
 		parsedURL, err := url.Parse(relayURL)
 		if err != nil {
-			log.Warn().Err(err).Str("url", relayURL).Msg("failed to parse relay URL")
+			log.Warn().Err(err).Str("url", relayURL).Msg("failed to parse relay URL, skipping")
 			continue
 		}
-
+		validRelayURLs = append(validRelayURLs, parsedURL.Host)
 		conn := NewConnection(identity, relayURL, session, twinID)
-		if conn.TryConnect() {
-			log.Info().Str("url", relayURL).Msg("connected")
-			successfulRelayURLs = append(successfulRelayURLs, parsedURL.Host)
-			successfulConnections = append(successfulConnections, conn)
-			continue
-		}
-
-		log.Warn().Str("url", relayURL).Msg("failed to connect")
+		connections = append(connections, conn)
 	}
 
-	if len(successfulRelayURLs) == 0 {
-		return nil, nil, errors.New("failed to connect to any relay")
+	if len(connections) == 0 {
+		return nil, nil, errors.New("no valid relay URLs provided")
```

Comment:
extract static errors into variables

Reply:
I have extracted the static error strings touched in this PR into package-level variables